src/deciderjob/agent/DeciderJobAgent.php

```diff
@@ -169,8 +169,15 @@ function processUploadId($uploadId)
 
     $this->licenseMap = new LicenseMap($this->dbManager, $this->groupId, $this->licenseMapUsage);
 
-    $this->processClearingEventOfCurrentJob();
-
+    if ($this->conflictStrategyId == 'global') {
+      if (!empty($this->clearingDao->getCandidateLicenseCountForCurrentDecisions(0, $uploadId))) {
```

Comment:
instead of passing 0 directly can you please assign to a variable and pass that variable.
that will increase the readability of the code.